exec/java-exec/src/main/java/org/apache/drill/exec/planner/PlannerPhase.java

```diff
@@ -180,9 +180,19 @@ public RuleSet getRules(OptimizerRulesContext context, Collection<StoragePlugin>
 
   PHYSICAL("Physical Planning") {
     public RuleSet getRules(OptimizerRulesContext context, Collection<StoragePlugin> plugins) {
+      final RuleSet storageRules = getStorageRules(context, plugins, this);
+
+      final Builder<RelOptRule> newStorageRulesBuilder = ImmutableSet.builder();
+      for (final RelOptRule relOptRule : storageRules) {
+        // exclude ProjectRemoveRule to prevent Drill from losing column alias or displaying implicit columns
+        if (!(relOptRule instanceof ProjectRemoveRule)) {
```

Comment:
For query  "select version as current_version from t;", why will ProjectRemoveRule think that the Project operator is a trivial project? I assume that the input/output's field name is different.

If there is bug in ProjectRemoveRule, it's better to fix that, in stead of removing it completely.